src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -175,12 +176,13 @@ export default {
 			}
 		},
 
-		isReadOnly() {
+		disabled() {
 			return this.conversation.readOnly === CONVERSATION.STATE.READ_ONLY
+			|| !this.currentConversationIsJoined
 		},
 
 		placeholderText() {
-			return this.isReadOnly
+			return this.disabled
```

Comment:
This is not correct. Now you replace the placeholder of the input with 'This conversation has been locked' while joining. So just copy the original value for this:
```suggestion
			return this.conversation.readOnly === CONVERSATION.STATE.READ_ONLY
```